subber/subber.cfg.example

```diff
@@ -2,5 +2,4 @@
 id=ID
 secret=SECRET
 password=PASSWORD
-username=USERNAME
-
+username=USERNAME
```

Comment:
Thanks @danieledwardknudsen! This commit removed the newline character from the end of `subber.cfg.example`. Would you mind restoring this?